scheduler/src/cook/mesos/util.clj

```diff
@@ -66,6 +66,7 @@
 (defonce ^Cache job-ent->resources-cache (new-cache))
 (defonce ^Cache categorize-job-cache (new-cache))
 (defonce ^Cache task-ent->user-cache (new-cache))
+(defonce ^Cache job-ent->user-cache (new-cache))
```

Comment:
I'm fine with making it a separate PR, but given that these caches are the reason we began talking about implementing something like mount, it would be good to follow through on that. Can we do the mount-ification PR right after this one?

Reply:
Sure, Lets get this out and then we can have that discussion. :)

Reply:
SGTM, do we have a Github issue to track this?